src/Models/HydrostaticFreeSurfaceModels/hydrostatic_free_surface_velocity_fields.jl

```diff
@@ -2,7 +2,7 @@ using Oceananigans.Grids: Center, Face
 using Oceananigans.Fields: XFaceField, YFaceField, ZFaceField
 using Oceananigans.BoundaryConditions: UVelocityBoundaryConditions, VVelocityBoundaryConditions, FieldBoundaryConditions
 
-function HydrostaticFreeSurfaceVelocityFields(arch, grid, bcs=NamedTuple())
+function HydrostaticFreeSurfaceVelocityFields(::Nothing, arch, grid, clock, bcs=NamedTuple())
     u_bcs = :u ∈ keys(bcs) ? bcs.u : UVelocityBoundaryConditions(grid)
```

Comment:
If this is every generalized to other models then it might be nice to generalize these so it works for all fields, but I don't think you need to worry about this for now